.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow will run tests using node and then publish a
+# package to GitHub Packages when a release is created
+# For more information see:
+# https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+      - run: npm test
+
+  publish-npm:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm publish
```

Comment:
How will we ensure that the version in package.json matches the tagged release?

Or won't we? Will this just be a manual thing?

Reply:
Perhaps `npm version` can solve this for us? (Although that would still be a manual step.)

Reply:
Good question! My intention - having got the "github actions" setup - is to switch the repo to `semantic-release` and let that take care of everything for us. So far I got as far as sorting out `commitizen` to make `git commit`-ing easier and conformative, and am now [mid] setting up `semantic-release`... which [as expected] doesn't have a Drone CI option, so the next best thing seems to be "github actions"...

Reply:
`semantic-release` is really interesting. It does seem complicated though.

I would be interested to see how it fares on a small project like this.

Reply:
Agreed. It's nice to have this little isolated repo so that we can trial it here. There have been a few bits to configure, but it basically pushes good practise and documentation into the commit messages (something we were not far off from anyway) so that everything can automatically flow from there, which makes a lot of sense to me.